README.md

```diff
@@ -10,7 +10,7 @@ SPDX-License-Identifier: Apache-2.0
 
 In addition to `sbomnix` this repository is a home to [nixgraph](./doc/nixgraph.md), a python library and command line utility for querying and visualizing dependency graphs for [nix](https://nixos.org/) packages.
```

Comment:
In addition to sbomnix this repository **is home** to nixgraph—a **P**ython library and a command line utility for querying and visualizing dependency graphs for **N**ix **P**ackages.